apis/swagger.yml

```diff
@@ -518,6 +518,29 @@ paths:
           $ref: "#/responses/500ErrorResponse"
       tags: ["Container"]
 
+  /containers/{id}/wait:
+    post:
+      summary: "Block until a container stops, then returns the exit code."
+      operationId: "ContainerWait"
+      parameters:
+        - $ref: "#/parameters/id"
+      responses:
+        200:
+          description: "The container has exited."
+          schema:
+            type: "object"
+            required: [StatusCode]
+            properties:
+              StatusCode:
+                description: "Exit code of the container"
+                type: "integer"
+                x-nullable: false
```

Comment:
Since the `ProbeContainer` maybe return the unexpected error, I think we should add the error in the response.

Reply:
So, the error means 500?  The error is caused by the container, not the server. The `waitContainer` API doesn't meets any error during the waiting. That is why that I suggest to use the error in the response.